src/components/home/homeMidSection/homeMidSection.tsx

```diff
@@ -107,9 +107,9 @@ const HomeMidSection = () => {
       badgeTitle: BadgeTitleGen(
         faDna,
         "Platforms: " +
-          summary.resource_stats.SequencingProtocol.stats.type.length.toString()
+        String(summary.resource_stats.SequencingProtocol !== undefined ? summary.resource_stats.SequencingProtocol?.stats.type.length.toString() : 0)
```

Comment:
```suggestion
        (summary?.resource_stats?.SequencingProtocol?.stats?.type?.length || 0)
```

You don't need to convert to string, JavaScript does this automatically.

Reply:
Similarly below.